exercises/hello-world/metadata.yml

```diff
@@ -1,4 +1,4 @@
 ---
-blurb: 'Greet the user by name, or by saying "Hello, World!" if no name is given.'
+blurb: 'Just say "Hello, World!"'
```

Comment:
I changed it, but I really don't know what would be good to put there.
If you have any suggestion, please write it here and I'll change it immediately.